pinna_bot:
  # Publish all joint states ----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 30  

  # Velocity controller ---------------------------------------
  wheelLeft_velocity_controller:
    type: velocity_controllers/JointVelocityController  
    joint: joint_wheel_left
    pid: {p: 100.0, i: 0.01, d: 10.0}

  wheelRight_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_wheel_right
    pid: {p: 100.0, i: 0.01, d: 10.0}
  
  #Interface twist_to_control parameter -----------------------
  twist_to_control_parameter:
    wheel_separation_multiplier: 1.07 #Dist betwen wheel MULTIPLIER
    wheel_radius_multiplier: 0.98 #Wheel radius MULTIPLIER